brage-import/src/main/java/no/sikt/nva/brage/migration/model/entitydescription/Contributor.java

```diff
@@ -0,0 +1,84 @@
+package no.sikt.nva.brage.migration.model.entitydescription;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+
+@JacocoGenerated
+public class Contributor {
+
+    private String type;
+    private Identity identity;
+    private String role;
+
+    private String brageRole;
+
+    public Contributor() {
+        
+    }
+
+    public Contributor(String type, Identity identity, String role, String brageRole) {
+        this.type = type;
+        this.identity = identity;
+        this.role = role;
+        this.brageRole = brageRole;
+    }
+
+    @JsonProperty("role")
+    public String getRole() {
+        return role;
+    }
+
+    public void setRole(String role) {
+        this.role = role;
+    }
+
+    @JsonProperty("type")
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    @JsonProperty("identity")
+    public Identity getIdentity() {
+        return identity;
+    }
+
+    public void setIdentity(Identity identity) {
+        this.identity = identity;
+    }
+
+    public String getBrageRole() {
+        return brageRole;
+    }
+
+    public void setBrageRole(String brageRole) {
+        this.brageRole = brageRole;
+    }
+
+    @JacocoGenerated
```

Comment:
I note you don't add @JacocoGenerated to every overridden method…is there a reason? Some classes don't have the overridden methods…I generally throw them in since I test equality in the tests

Reply:
why do you add jacocogenerated when you dont need it?